CHANGELOG

```diff
@@ -1,5 +1,6 @@
 # 3.15.0 (2024-XX-XX)
 
+ * Deprecate the `attribute` function; use the `.` notation and wrap the name with parenthesis instead
```

Comment:
Deprecating the `attribute` function would make it hard to write Twig templates supporting older Twig versions (we cannot do a `{% if %}` to switch between 2 cases, we have to switch between 2 templates instead as the deprecation is at compile time).

As the `attribute` function is free to maintain (the implementation uses the same PHP function than the `.` operator), I suggest delaying the deprecation until some future 4.x release (we have some maintained versions of Symfony that still support Twig 2.x and so could not upgrade).

Reply:
That works for me. Code updated accordingly.

Reply:
See 8d1af0fc03314a2786decd6457447a583f7dd146